content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -71,6 +71,10 @@ An _output entry_ (`outs`) consists of these fields:
   _checksum_ for HDFS.
 - `cache`: Whether or not this file or directory is <abbr>cached</abbr> (`true`
   by default, if not present). See the `--no-commit` option of `dvc add`.
+- `backup`: Whether or not this file or directory is backed-up to project
+  remotes. (`true` by default for all `outs` except imports, where it is `false`
+  by default). Mostly relevant for imports in
```

Comment:
Can it be set to `false` for regular .dvc files? (Non-imports)

Reply:
I feel like it should probably only have an effect for imports (and if I;m right, we can say so here 🙂)

Reply:
@efiop and I had a bit of a chat about this already - see https://github.com/iterative/dvc/issues/4527 for a summary. 

My thinking is that it's good to try and avoid special treatment for particular types of file where possible: it's better to have the behaviour explicit in the .dvc file. The code PR https://github.com/iterative/dvc/pull/4581 really removes special treatment for `outs` which result from imports vs `outs` which result from adds or pipelines: the difference is now that the former have `backup: false` whereas the latter have `backup: true`. To me it seems odd that something in the `deps` part of the file would affect how the `outs` are treated, but that's what you have to do if you want to treat imports as special cases. 

It would also be surprising for the user if you had `push: false` in a normal dvc file but which still resulted in that file getting pushed to the remote. And hey, if someone wants a normal dvc added file which doesn't get pushed to remotes, why shouldn't we let them have one?

Reply:
> it seems odd that something in the deps part of the file would affect how the outs are treated, but that's what you have to do if you want to treat imports as special cases

Yeah OK, makes sense. From an implementation perspective at least

> surprising for the user if you had push: false in a normal dvc file but which still resulted in that file getting pushed

But DVC would never add backup:false to regular .dvc files so users doing it manually should also be an edge case (in which case they should know about the field and the docs would explain it only affects imports)? But anyway, let's just document the way it works for now, and maybe I'll jump into the discussions in the core repo.

> if someone wants a normal dvc added file which doesn't get pushed to remotes, why shouldn't we let them have one?

Good point.

---
BTW:

> The idea is for imports with --backup to be severed from their import source completely unless the user calls dvc update explicitly.

- [ ] Great explanation in that ticket! We should probably include this sentence more or less as is in the docs 🙂

Reply:
p.s. I see now you were trying to add that info about `update` but I removed it haha sorry — it didn't seem necessary the way it was written. I'll re-add it.